src/Core/Usecase/User/LoginUser.php

```diff
@@ -16,6 +16,9 @@
 use Ushahidi\Core\Usecase\ReadRepository;
 use Ushahidi\Core\Usecase\ReadUsecase;
 
+use BehEh\Flaps\Flap;
```

Comment:
Core Usecases shouldn't rely on anything outside Ushahidi\Core directly.
We can still use these, but they should be wrapped in a Core interface / object.

Ideally something super simple like an `Ushahidi\Core\Tools\RateLimiter` interface with just a `limit(Entity $entity)` method. Then extend Flap to implement that interface and forward the entity id to Flap.